### ISSUE TITLE
Custom Ped Outfit.

### ISSUE THREAD

**Issue by setup2fail** [FrenzyRoleplay/server-issues#588]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

So, basically ever since I've started rping in the last few years I've not just wanted but NEEDED to role-play as an overweight officer lol. But the only way to do that is using an overweight ped. The problem is they have specific outfits and they can't use the outfits non peds can (as I'm sure you know) So, I was wondering if I picked out a character (I don't have a specific one in mind yet, as I wanted to find out if it's even possible first.) If someone from your wonderful team of devs would be able to make a custom outfit for sheriff or lspd or both. I'd like to be In sheriff but I don't know how the process works. My app was already accepted (I have a character made but not fat I will just make a new one but use the same backstory if this happens..
 except with me fat which actually works because my character was overweight as a child.) I still have to do the interview though for DOC. But I know what I'm doing I've rp'd as a leo before so I'm confident I will get hired. I'd even love to have a corrections outfit as well because I may be there a while waiting for the process to move to cadet. ALL I've ever wanted to do since rping is this, I've seen its possible in some cities and knowing how amazing your city is I figured there's gotta be someone talented enough and willing to do it. I will literally pay cash. I don't have a lot but I'd be willing to pay like 100$ U.S for it to be done. I promise if yall were able to make this happen you would love my rp as an overweight officer. Let me know, either way thanks so much for all you guys do.

**Comment by setup2fail** [comment]
Not to rush anyone I know yall are busy working hard but I kinda need an answer before I can do my leo interview

**Comment by ProximateTv** [comment]
I have received a similar request from two other people. Maybe I can do a one size fits all type deal and you all can share the dream or something.

**Comment by setup2fail** [comment]
Yeah let me know, like I said I'd gladly pay your for your time/work. I think there would be quite a few people who would enjoy the rp from it.
